wellcomeml/datasets/winer.py

```diff
@@ -0,0 +1,222 @@
+# WiNER: A Wikipedia Annotated Corpus for Named Entity Recognition
+# https://www.aclweb.org/anthology/I17-1042/
+
+from wellcomeml.datasets.download import check_cache_and_download
+
+# def load_winer():
+import tarfile
+import os
+from tqdm import tqdm 
+import json
+import random
+
+def create_train_test(
+    NE_path, vocab_path, docs_path,
+    train_processed_path, test_processed_path,
+    n_sample, prop_train, rand_seed=42
+    ):
+    """
+    NE_path: File path to the named entities
+    vocab_path: File path to the vocab dictionary
+    docs_path: File path to the documents texts (words coded according to the vocab)
+    train_processed_path: Output file path for the training data
+    test_processed_path: Output file path for the testing data
+    n_sample: Number of files to get data from, each file contains about 1000 documents
+    prop_train: Proportion of the file sample that will be in the training data
+
+    Utilise the three datasources to get the entities from a sample
+    of documents and save to a training and testing dataset files
+    in the form:
+
+    ID 12345
+    John Smith PERSON
+    rejects O
+    German LOC
+    call O
+    
+    Next 0
+    sentence 0
+
+    ID 444
+    Next
+    document
+    """
+
+    random.seed(rand_seed)
+    n_train = round(prop_train*n_sample)
+
+    print("Creating entities dictionary from {}".format(NE_path))
+    entities = {}
+    with tarfile.open(NE_path, "r:bz2") as tar:
+        for member in tqdm(tar.getmembers()):
+            f = tar.extractfile(member)
+            if f:
+                content = f.read().decode('utf-8', errors='ignore')
```

Comment:
are you sure ignorning is the right behavior? don't they mention the right format somewhere?